node.go

```diff
@@ -319,3 +320,25 @@ func (n *Node) SendBlock(address string, blockNode *BlockNode) {
 		log.Fatal(err)
 	}
 }
+
+func (n *Node) TryToFindNewBlock() {
+	for {
+		transaction := []Transaction{
+			{Sender: []byte("Alice"), Receiver: []byte("Bob"), Amount: 10},
+		}
+		newBlock := n.blockchain.GenerateNewBlock(transaction)
+		nonce := uint64(0)
+
+		for {
+			newBlock.SetData(nonce)
+			parentBlock := n.blockchain.GetLatestBlock()
+			if err := n.blockchain.ValidateBlock(newBlock, parentBlock); err == nil {
+				break
+			}
+			nonce++
+		}
+
+		n.blockchain.AddBlock(n.blockchain.root, newBlock)
+		time.Sleep(10 * time.Second)
```

Comment:
instead of sleeping here, i would increase number of zeros required in block hash, so it would require more CPU time to find new best block